web/gds-user-ui/src/modules/dashboard/member/UnverifiedMember.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box, Text } from '@chakra-ui/react';
+import { Trans } from '@lingui/macro';
+
+const UnverifiedMember = () => {
+  return (
+    <Box width={'100%'} bg={'#F7F8FC'} p="8" mx={'auto'}>
+      <Text fontSize="md" color="gray.700" fontWeight="bold" mb={2}>
```

Comment:
This could be a nitpick, but in Figma the text color is black and the font-weight isn't bold.

Reply:
I appreciate your feedback. I have made the text bold to ensure consistency with the other alert components. This change enhances the visibility of the notice, allowing users to quickly identify the component.
I will leave a comment to @eschmier to let him about that when he will be back.

Reply:
I just noticed that the content is not centered, I just add it.